From d74f9fc25a5badaccabbdbabb09b6f95a61324c8 Mon Sep 17 00:00:00 2001
From: Giovanni Ricca <giovanniricca@proton.me>
Date: Mon, 12 Jan 2026 13:53:50 +0100
Subject: [PATCH] dodge: overlay: Restore old paddings

Reason: They look better imho.

This reverts commits e33d6f7268468f6aad15e3741d19ff17e1ea322b and 92b7b3a982d7d8180b96ba7a4f17aa9e0c60e5b6.

Change-Id: I4bdedf0198bdcedd7570de5f5b9e62fedd88bb35
---
 .../res/values-land/dimens.xml                   | 15 +++++++++++++++
 .../res/values-sw320dp/dimens.xml                | 16 ----------------
 .../res/values-sw360dp/dimens.xml                | 16 ----------------
 .../res/values-sw411dp/dimens.xml                | 16 ----------------
 .../res/values/dimens.xml                        | 13 ++++++-------
 .../res/drawable/rounded.xml                     | 14 ++++++++++++++
 .../res/values-land/dimens.xml                   | 15 +++++++++++++++
 .../OPlusSystemUIResTarget/res/values/dimens.xml | 14 +++++++++++++-
 8 files changed, 63 insertions(+), 56 deletions(-)
 create mode 100644 overlay/OPlusFrameworksResTarget/res/values-land/dimens.xml
 delete mode 100644 overlay/OPlusFrameworksResTarget/res/values-sw320dp/dimens.xml
 delete mode 100644 overlay/OPlusFrameworksResTarget/res/values-sw360dp/dimens.xml
 delete mode 100644 overlay/OPlusFrameworksResTarget/res/values-sw411dp/dimens.xml
 create mode 100644 overlay/OPlusSystemUIResTarget/res/drawable/rounded.xml
 create mode 100644 overlay/OPlusSystemUIResTarget/res/values-land/dimens.xml

diff --git a/overlay/OPlusFrameworksResTarget/res/values-land/dimens.xml b/overlay/OPlusFrameworksResTarget/res/values-land/dimens.xml
new file mode 100644
index 0000000..b403d01
--- /dev/null
+++ b/overlay/OPlusFrameworksResTarget/res/values-land/dimens.xml
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     SPDX-FileCopyrightText: The LineageOS Project
+     SPDX-License-Identifier: Apache-2.0
+-->
+<resources>
+    <!-- The default height of the status bar used in {@link SystemBarUtils#getStatusBarHeight} to
+         calculate the status bar height. -->
+    <dimen name="status_bar_height_default">@dimen/status_bar_height</dimen>
+
+    <!-- Height of the status bar.
+         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
+         -->
+    <dimen name="status_bar_height">98px</dimen>
+</resources>
diff --git a/overlay/OPlusFrameworksResTarget/res/values-sw320dp/dimens.xml b/overlay/OPlusFrameworksResTarget/res/values-sw320dp/dimens.xml
deleted file mode 100644
index b15f356..0000000
--- a/overlay/OPlusFrameworksResTarget/res/values-sw320dp/dimens.xml
+++ /dev/null
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-     SPDX-FileCopyrightText: The LineageOS Project
-     SPDX-License-Identifier: Apache-2.0
--->
-<resources>
-    <!-- Height of the status bar in portrait.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_portrait">35.56dp</dimen>
-
-    <!-- Height of the status bar in landscape.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_landscape">35.56dp</dimen>
-</resources>
diff --git a/overlay/OPlusFrameworksResTarget/res/values-sw360dp/dimens.xml b/overlay/OPlusFrameworksResTarget/res/values-sw360dp/dimens.xml
deleted file mode 100644
index 89cc31d..0000000
--- a/overlay/OPlusFrameworksResTarget/res/values-sw360dp/dimens.xml
+++ /dev/null
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-     SPDX-FileCopyrightText: The LineageOS Project
-     SPDX-License-Identifier: Apache-2.0
--->
-<resources>
-    <!-- Height of the status bar in portrait.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_portrait">40dp</dimen>
-
-    <!-- Height of the status bar in landscape.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_landscape">40dp</dimen>
-</resources>
diff --git a/overlay/OPlusFrameworksResTarget/res/values-sw411dp/dimens.xml b/overlay/OPlusFrameworksResTarget/res/values-sw411dp/dimens.xml
deleted file mode 100644
index 2983faa..0000000
--- a/overlay/OPlusFrameworksResTarget/res/values-sw411dp/dimens.xml
+++ /dev/null
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-     SPDX-FileCopyrightText: The LineageOS Project
-     SPDX-License-Identifier: Apache-2.0
--->
-<resources>
-    <!-- Height of the status bar in portrait.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_portrait">45.67dp</dimen>
-
-    <!-- Height of the status bar in landscape.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_landscape">45.67dp</dimen>
-</resources>
diff --git a/overlay/OPlusFrameworksResTarget/res/values/dimens.xml b/overlay/OPlusFrameworksResTarget/res/values/dimens.xml
index 947e199..f363329 100644
--- a/overlay/OPlusFrameworksResTarget/res/values/dimens.xml
+++ b/overlay/OPlusFrameworksResTarget/res/values/dimens.xml
@@ -4,16 +4,15 @@
      SPDX-License-Identifier: Apache-2.0
 -->
 <resources>
-    <!-- Height of the status bar in portrait.
-         Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
-         -->
-    <dimen name="status_bar_height_portrait">40dp</dimen>
+    <!-- The default height of the status bar used in {@link SystemBarUtils#getStatusBarHeight} to
+         calculate the status bar height. -->
+    <dimen name="status_bar_height_default">@dimen/status_bar_height</dimen>
 
-    <!-- Height of the status bar in landscape.
+    <!-- Height of the status bar.
          Do not read this dimen directly. Use {@link SystemBarUtils#getStatusBarHeight} instead.
          -->
-    <dimen name="status_bar_height_landscape">40dp</dimen>
+    <dimen name="status_bar_height">110px</dimen>
 
     <!-- Default radius of the software rounded corners. -->
-    <dimen name="rounded_corner_radius">112px</dimen>
+    <dimen name="rounded_corner_radius">8dp</dimen>
 </resources>
diff --git a/overlay/OPlusSystemUIResTarget/res/drawable/rounded.xml b/overlay/OPlusSystemUIResTarget/res/drawable/rounded.xml
new file mode 100644
index 0000000..1dd3baa
--- /dev/null
+++ b/overlay/OPlusSystemUIResTarget/res/drawable/rounded.xml
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="209px"
+    android:height="209px"
+    android:viewportWidth="209"
+    android:viewportHeight="209">
+
+    <path
+        android:fillColor="#ff000000"
+        android:pathData="M196.755,-0L-0,-0L-0,208.273C-0,116.958 2.154,70.516 39.062,36.485C75.202,3.163 108.538,-0 196.755,-0Z"
+        android:strokeColor="#00000000"
+        android:strokeWidth="1.0"
+        android:fillType="evenOdd" />
+
+</vector>
diff --git a/overlay/OPlusSystemUIResTarget/res/values-land/dimens.xml b/overlay/OPlusSystemUIResTarget/res/values-land/dimens.xml
new file mode 100644
index 0000000..c4d6425
--- /dev/null
+++ b/overlay/OPlusSystemUIResTarget/res/values-land/dimens.xml
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     SPDX-FileCopyrightText: The LineageOS Project
+     SPDX-License-Identifier: Apache-2.0
+-->
+<resources>
+    <!-- the padding on the start of the statusbar -->
+    <dimen name="status_bar_padding_start">63px</dimen>
+
+    <!-- the padding on the end of the statusbar -->
+    <dimen name="status_bar_padding_end">33px</dimen>
+
+    <!-- the padding on the top of the statusbar (usually 0) -->
+    <dimen name="status_bar_padding_top">4px</dimen>
+</resources>
diff --git a/overlay/OPlusSystemUIResTarget/res/values/dimens.xml b/overlay/OPlusSystemUIResTarget/res/values/dimens.xml
index 4f22b92..621746d 100644
--- a/overlay/OPlusSystemUIResTarget/res/values/dimens.xml
+++ b/overlay/OPlusSystemUIResTarget/res/values/dimens.xml
@@ -5,7 +5,19 @@
 -->
 <resources>
     <!-- the padding on the start of the statusbar -->
-    <dimen name="status_bar_padding_start">8dp</dimen>
+    <dimen name="status_bar_padding_start">60px</dimen>
+
+    <!-- the padding on the end of the statusbar -->
+    <dimen name="status_bar_padding_end">16px</dimen>
+
+    <!-- the padding on the top of the statusbar (usually 0) -->
+    <dimen name="status_bar_padding_top">16px</dimen>
+
+    <!-- Height of the status bar header bar when on Keyguard -->
+    <dimen name="status_bar_header_height_keyguard">@*android:dimen/status_bar_height</dimen>
+
+    <!-- Margin on the left side of the carrier text on Keyguard -->
+    <dimen name="keyguard_carrier_text_margin">@dimen/status_bar_padding_start</dimen>
 
     <!-- Location on the screen of the center of the physical power button. This is a reasonable
     default that should be overridden by device-specific overlays. -->
-- 
2.52.0

